README.md

```diff
@@ -20,13 +20,13 @@ Blog post for the GA feature can be found [here](https://kubernetes.io/blog/2020
 
 This information reflects the head of this branch.
 
-| Compatible with CSI Version                                                                | Container Image             | [Min K8s Version](https://kubernetes-csi.github.io/docs/kubernetes-compatibility.html#minimum-version) |
-| ------------------------------------------------------------------------------------------ | ----------------------------| --------------- |
-| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | k8s.gcr.io/sig-storage/csi-snapshotter | 1.17         |
-| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | k8s.gcr.io/sig-storage/snapshot-controller  | 1.17     |
-| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | k8s.gcr.io/sig-storage/snapshot-validation-webhook  | 1.17     |
+| Supported CSI Versions                                                                | Container Image             | [Min K8s Version](https://kubernetes-csi.github.io/docs/kubernetes-compatibility.html#minimum-version) | [Recommended K8s Version](https://kubernetes-csi.github.io/docs/project-policies.html#recommended-version)
+| ------------------------------------------------------------------------------------------ | ----------------------------| --------------- | --------------- |
+| [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0), [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0), [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0), [CSI Spec v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0), [CSI Spec v1.4.0](https://github.com/container-storage-interface/spec/releases/tag/v1.4.0) | k8s.gcr.io/sig-storage/csi-snapshotter | 1.20         | 1.20         |
```

Comment:
Do we need to mention all of the version? I think just mentioning 1.0 is min, and 1.4 as recommended is sufficient

Reply:
Sure.